features/buyers/account_plans/index.feature

```diff
@@ -51,20 +51,22 @@ Feature: Account plans index page
 
     Scenario: Copying account plans
       When they select action "Copy" of "Public Plan"
-      And they select action "Copy" of "Secret Plan"
-      And wait a moment
       Then they should see "Plan copied"
       And the table has the following rows:
         | Name               | Contracts | State     |
         | Public Plan (copy) | 0         | published |
+
+      When they select action "Copy" of "Secret Plan"
+      Then they should see "Plan copied"
+      And the table has the following rows:
+        | Name               | Contracts | State     |
         | Secret Plan (copy) | 0         | hidden    |
 
     Scenario: Deleting account plans
       Given account plan "Public Plan" has 0 contracts
       And they go to the account plans admin page
       When they select action "Delete" of "Public Plan"
       And confirm the dialog
-      And wait a moment
```

Comment:
hopefully avoid bad timing where message would disappear while "waiting a moment"

https://app.circleci.com/pipelines/github/3scale/porta/27372/workflows/d0505ef3-699f-49d0-8923-ddbcd1f92210/jobs/306904/tests

I think that we don't need to wait a moment because `they should see` I assume uses capybara waiting under the hood.

Reply:
Or not? I'm not sure what `page.should have_content(regex)` does.

Reply:
Yeah, in theory both should wait...